static/fix-links.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+document.addEventListener("DOMContentLoaded", () => {
+  const production_base_url = new URL("https://backoffice.seattleflu.org/switchboard");
+
+  /* Fix broken links generated by Datasette when deployed at a subpath instead
+   * of the web root.  See <https://github.com/simonw/datasette/issues/838>.
+   * This is kinda terrible, but at least it makes the links work.  It seemed
+   * more palatable than configuring Apache (our reverse proxy) to rewrite all
+   * HTML from Datasette using mod_proxy_html.
+   *   -trs, 2 July 2020
+   */
+  if (document.location.origin !== production_base_url.origin) {
+    console.debug(`fix-links: skipping page; origin ${document.location.origin} is not ${production_base_url.origin}`)
+    return;
+  }
+
+  const broken_link = url =>
+       url.origin !== production_base_url.origin
+    && url.origin.match(/localhost|127[.]0[.]0[.]1/);
```

Comment:
I'm curious, why do you use square brackets here to "escape" the `.` instead of using a backslash? e.g. `127\.0\.0\.1`

Reply:
Good question! No particular reason. I started using `[.]` in the last year or so in situations where backslashes are annoying to escape, and it's bled over into more general situations since then.